### PR TITLE
Download links and nextpush and new instructions

### LINKED ISSUE #39: Add link to Izzi On Droid repo
https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it
https://apt.izzysoft.de/fdroid/index/apk/org.unifiedpush.distributor.fcm
https://apt.izzysoft.de/fdroid/index/apk/org.unifiedpush.distributor.nextpush

### LINKED ISSUE #40: Add nextpush page and be more distributor neutral
Most of the recommendations are Gotify, NextPush might actually be a better choice for some people and that should be presented to new users properly.

### PR DESCRIPTION
**Pull request by karmanyaahm**
- Add NextPush distributor page
- Change Links to badges

Closes #39 Closes #40

### CODE REVIEW

**Comment by karmanyaahm** [comment]
I also made changes that I hope will be slightly more helpful to explain UP to new users. Comments are welcome :)

File: content/users/intro.md [hunk 1]
```diff
@@ -3,15 +3,53 @@ title: Introduction
 weight: -10
 ---
 
-## Definitions TLDR
+## What is UnifiedPush?
 
+**What are push notifications?** Your mobile device often needs to receive updates from many messaging, and sometimes other apps that provide real-time updates. Without push notifications, each app would need to contact its own server to get updates. This wastes a lot of battery power. So, push notifications were created; they allow you to receive updates from multiple apps using one network connection by routing everything through a single server. 
+
+UnifiedPush is a standard for receiving these push notifications.
+UnifiedPush allows you to choose between multiple implementations and servers, based on your preferences. The core UnifiedPush tools are all free and open source software.
```
Comment (p1gp1g) [inline content/users/intro.md:11]: Wouldn't it be better on the root index ?

File: content/users/intro.md [hunk 2]
```diff
@@ -3,15 +3,53 @@ title: Introduction
...
+1. Install [the Gotify-UP app](../distributors/gotify)   
+    Note: `Gotify-UP` is different from `Gotify` as the latter doesn't support UnifiedPush.
+2. Visit <https://gotify1.unifiedpush.org> and click on `Register`. Sign up with a username and secure password.
+3. Open the Gotify-UP app on your device, and login with those credentials.
+4. Now, you should be able to open any [apps supporting UnifiedPush](../apps), and should be able to receive push notifications!
+  * Note: if an app asks you which push distributor you want, you can now pick Gotify (though it probably shouldn't ask you).
+
+### Difficult - Self Host the server
+
+The lets you have control of the server through which you receive push notifications. You have several distributor/provider options for self-hosting:
```
Comment (p1gp1g) [inline content/users/intro.md:31]: It looks like this sentence isn't finished

File: content/users/intro.md [hunk 3]
```diff
@@ -3,15 +3,53 @@ title: Introduction
...
-* The (Push) **Provider** is the server reachable from the Internet that listens for incoming notifications for you. The Distributor is connected to the Provider to get the notifications.
+* The (Push) **Provider** is the server reachable from the Internet that listens for incoming notifications for you.
+* The **Distributor** is connected to the **Provider** to get the notifications.
+
+## Install a Distributor
+
+### Easy - Use UnifiedPush hosted server
+
+1. Install [the Gotify-UP app](../distributors/gotify)   
+    Note: `Gotify-UP` is different from `Gotify` as the latter doesn't support UnifiedPush.
```
Comment (p1gp1g) [inline content/users/intro.md:23]: It misses `  *` I think

**Review by p1gp1g — COMMENTED** [review]
It looks nice :)

File: content/users/intro.md (hunk 1 above)
Comment (karmanyaahm) [reply content/users/intro.md:11]: I feel like putting it on this page makes it so new users just have to read one small page which might be better for retention. However, it makes sense to put it on the root so I'll put it in both?

File: content/users/intro.md (hunk 2 above)
Comment (karmanyaahm) [reply content/users/intro.md:31]: The colon `:` is just a prefix for a general list, i think (in this case the following list of distributors)

**Comment by karmanyaahm** [comment]
Also, @p1gp1g do you want to mark NextPush WIP here as it is on the repos? I can change that in the PR if so, or if not, no changes are needed for that

File: content/users/intro.md (hunk 1 above)
Comment (p1gp1g) [reply content/users/intro.md:11]: sounds ok !

**Comment by p1gp1g** [comment]
> Also, @p1gp1g do you want to mark NextPush WIP here as it is on the repos? I can change that in the PR if so, or if not, no changes are needed for that

I've marked them beta. We can tell it is in beta on the doc and removing it when it will be tested enough

File: content/users/intro.md (hunk 2 above)
Comment (karmanyaahm) [reply content/users/intro.md:31]: oh oops i missed that :laughing:

**Comment by karmanyaahm** [comment]
@p1gp1g it should be ready to merge now

**Pull request merged**